scipy/special/orthogonal.py

```diff
@@ -1344,7 +1349,13 @@ def roots_gegenbauer(n, alpha, mu=False):
         # keep doing so.
         return roots_chebyt(n, mu)
 
-    mu0 = np.sqrt(np.pi) * cephes.gamma(alpha + 0.5) / cephes.gamma(alpha + 1)
+    if alpha <= 170:
+        mu0 = np.sqrt(np.pi) * cephes.gamma(alpha + 0.5) / cephes.gamma(alpha + 1)
```

Comment:
And here it would be real easy to use Lanczos; though you could also cast it in terms of `sc.poch`.